modules/utils/docker/qos_olsrd_conf.sh

```diff
@@ -0,0 +1,89 @@
+
+#! /bin/bash
+
+
+create_dhcpd_config()
```

Comment:
How DHCP spoofing is prevented?

Reply:
It's not, I guess? However I believe with the move to SLAAC (will be another PR) then we will no longer be using DHCP. @TIISR am I correct?